core/src/main/java/org/apache/carbondata/core/datastore/filesystem/AlluxioCarbonFile.java

```diff
@@ -57,6 +57,15 @@ public AlluxioCarbonFile(FileStatus fileStatus) {
     return files;
   }
 
+  @Override
+  public String getAbsolutePath() {
```

Comment:
Is this PR enhance carbon on Alluxio also?

Reply:
will move to another pr